packages/app/src/containers/ExportInspector.tsx

```diff
@@ -0,0 +1,68 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import { Selectors } from 'noya-state';
+import { memo, useCallback } from 'react';
+import { useSelector, useDispatch } from '../contexts/ApplicationStateContext';
+import useShallowArray from '../hooks/useShallowArray';
+import withSeparatorElements from 'noya-designsystem/src/utils/withSeparatorElements';
+import { Divider } from 'noya-designsystem';
+import type FileFormat from '@sketch-hq/sketch-file-format-ts';
+import ArrayController from '../components/inspector/ExportArrayController';
+import ExportFormatsRow from '../components/inspector/ExportFormatsRow';
+import ExportPreviewRow from '../components/inspector/ExportPreviewRow';
+
+export default memo(function ExportInspector() {
+  const title = 'Exports';
+  const dispatch = useDispatch();
+
+  const selectedLayer = useShallowArray(
```

Comment:
We don't need `useShallowArray` here, since we're not passing this to the dependencies array of any hooks